src/client/game/conundrum-round/conundrum-results.js

```diff
@@ -1,15 +1,61 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import DDoris from './ddoris';
+
 import style from '../game.scss';
+import bannerStyles from './conundrum-round.scss';
 
 export class ConundrumResults extends Component {
+    componentWillReceiveProps(nextProps) {
+        if(nextProps.conundrumResults !== this.props.conundrumResults) {
+            this.conundrumTimer = setTimeout(this.flipLetters.bind(this), 500);
+        }
+    }
+
+    flipLetters() {
+        for(var i=0; i< this.props.conundrumResults.solution.length; i++){
+            this.refs[`letterBox${i}`].className += this.refs[`letterBox${i}`].className.includes(style.flipped) ? '' : ` ${style.flipped}` ;
```

Comment:
I'm not keen on using refs - they're meant to be deprecated a little, there's probably a different way to do this. Let's have a look together.